src/child/index.js

```diff
@@ -20,14 +21,21 @@ require('script!../../node_modules/BitFlux/dist/bitflux.js');
 /* eslint-enable import/no-unresolved */
 
 fin.desktop.main(() => {
-    fin.desktop.Window.getCurrent().contentWindow.addEventListener('beforeunload', () => {
-        fin.desktop.Window.getCurrent().contentWindow.opener.store.dispatch(close());
+    const store = fin.desktop.Window.getCurrent().contentWindow.opener.store;
+    fin.desktop.Window.getCurrent().contentWindow.store = fin.desktop.Window.getCurrent().contentWindow.opener.store;
+
+    $(window).unload(() => {        // eslint-disable-line no-undef
```

Comment:
What's the reason for using jQuery and changing from `beforeunload` to `unload`?